### DISCUSSION TITLE
Production Ready Advice

### DISCUSSION THREAD

**Discussion by aport-305** [NUWCDIVNPT/stigman-orchestration#13] [Q&A]
Ive been able to get stig manager up in running in a lab type of environment no problem and accessing the app from a localhost browser. In the instructions it clearly states the example docker-compose files aren't prod ready: "The example supports connections to and from localhost only and is NOT intended for production use.". Besides replacing localhost certs with newly generated cers what other changes should be made to have a "prod ready" deployment. I appreciate any advice!

**Comment by cd-rite** [comment]
Hi @aport-305 We can't really say what requirements your organization may have for production systems, but running with your own certificates (and ensuring they would be accepted by any client you want to support) would certainly be part of it! Beyond that, there are usually requirements around backing up your database, account management for your authentication provider, log aggregation, etc.  There are lots of different ways to accomplish this that are beyond the scope of the UI and API we offer in the project, which is why we include that note.
